pluto/src/vis/schematic/registry.ts

```diff
@@ -454,7 +455,7 @@ const reliefValve: Spec<ReliefValveProps> = {
     ...zeroLabel("Relief Valve"),
     ...ZERO_DUMMY_TOGGLE_PROPS,
   }),
-  Preview: ({ clickable, ...props }) => Primitives.ReliefValve(props),
```

Comment:
You have to make sure you don't pass `clickable` to `Primitives.ReliefValve`, otherwise an error for an unknown DOM property gets logged in the console. This should be fixed for all of the other ones too.